Gems/Atom/RPI/Code/Include/Atom/RPI.Public/RenderPipeline.h

```diff
@@ -113,6 +113,10 @@ namespace AZ
             //! It's the same as SetPersistentView(GetMainViewTag(), view)
             void SetDefaultView(ViewPtr view);
 
+            //! Set a stereoscopic view to the default view tag.
+            //! It's the same as SetPersistentView(GetMainViewTag(), view)
+            void SetDefaultStereoscopicViewFromEntity(EntityId entityId, uint32_t viewIndex);
```

Comment:
It's worth to mention viewIndex 0 is left and viewIndex 1 is right or add an enum for it?